grammar/JavaScript.tmLanguage.json

```diff
@@ -1,11 +1,14 @@
 {
-	"name": "JavaScript (with React support)",
-	"scopeName": "source.js",
+	"name.$comment": "** Changed **",
+	"name": "Mongo Scrapbooks (JavaScript)",
+	"scopeName.$comment": "** Changed **",
+	"scopeName": "source.mongo.js",
 	"fileTypes": [
 		".js",
```

Comment:
Worth looking into the 'fileTypes' property here? Not sure what it does...

Reply:
Changed to .mongo, though not sure if it has any affect.